hca/query/cli.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sys
+
+from . import DCPQueryClient
+
+def add_commands(subparsers, help_menu=False):
+    query_parser = subparsers.add_parser('query', help="Interact with the HCA DCP Query Service")
+
+    def help(args):
+        query_parser.print_help()
+
+    if sys.version_info >= (2, 7, 9):  # See https://bugs.python.org/issue9351
```

Comment:
Do we need to worry about python < `2.7.9`?

Reply:
See https://github.com/HumanCellAtlas/dcp-cli/issues/112